src/engine/enginebuffer.cpp

```diff
@@ -1193,24 +1199,35 @@ void EngineBuffer::processSeek(bool paused) {
     }
 
     if (!paused && (seekType & SEEK_PHASE)) {
-        if (kLogger.traceEnabled()) {
+        if (kLogger.traceEnabled())
             kLogger.trace() << "EngineBuffer::processSeek Seeking phase";
-        }
         double requestedPosition = position;
         double syncPosition = m_pBpmControl->getBeatMatchPosition(position, true, true);
         position = m_pLoopingControl->getSyncPositionInsideLoop(requestedPosition, syncPosition);
-        if (kLogger.traceEnabled()) {
+        if (kLogger.traceEnabled())
             kLogger.trace()
                     << "EngineBuffer::processSeek seek info: " << m_filepos_play
                     << " -> " << position;
-        }
     }
     if (position != m_filepos_play) {
-        if (kLogger.traceEnabled()) {
+        if (kLogger.traceEnabled())
             kLogger.trace() << "EngineBuffer::processSeek Seek to" << position;
+        if (m_macroRecordingState == MacroState::Armed) {
+            VERIFY_OR_DEBUG_ASSERT(m_pEngineMaster->m_pMacroRecording != nullptr) {
+                return;
+            }
+            m_pEngineMaster->m_pMacroRecording->appendHotcueJump(m_filepos_play, position);
```

Comment:
These are in units of interleaved stereo samples, right? If so, then you need to divide by mixxx::kEngineChannelCount to get frame positions.

Reply:
I have no idea tbh ^^

Reply:
Well, look into it and figure out if we should convert to frames here.

Reply:
Okay, so interleaved stereo samples means that the samples/frames of each (stereo) channel alternate, which means by dividing by the amount of channels I get the position on the mono track, right?
So yes, I should convert.

Reply:
Yes, that's correct.